src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/CommentEnteredSubscription.tsx

```diff
@@ -99,10 +99,70 @@ function determineDepthTillAncestor(
   return depth;
 }
 
+function getFlattenedReplyAncestorID(comment: RecordProxy, depth: number) {
+  const iterations = depth - 3;
```

Comment:
yeah once again the 3 here feels arbitrary

Reply:
👍